translate/api/Tests/BatchTranslateWithGlossaryTests.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Xunit;
+
+namespace GoogleCloudSamples
+{
+    [Collection(nameof(TranslateFixture))]
+    public class BatchTranslateWithGlossaryTests
+    {
+        private readonly string _inputUri = "gs://cloud-samples-data/translation/text_with_glossary.txt";
+        private readonly TranslateFixture _fixture;
+
+        private readonly CommandLineRunner _sample = new CommandLineRunner()
```

Comment:
I think you can add this to the fixture even, and remove it from all the tests.
```
public CommandLineRunner SampleRunner { get; } = new CommandLineRunner()
{
    VoidMain = TransalateV3Samples.Main
};
```

Reply:
Some tests dont need to create bucket & glossary so that I did not apply this change to tests with no bucket and glossary use.